app/src/main/java/com/paypal/psix/activities/SettingsActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.paypal.psix.activities;
+
+import android.os.Bundle;
+import android.preference.Preference;
+import android.preference.PreferenceActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import com.paypal.psix.R;
+
+public class SettingsActivity extends PreferenceActivity
+        implements Preference.OnPreferenceChangeListener {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        addPreferencesFromResource(R.xml.pref_general);
+
+        setupToolbar();
+        setupSignOut();
+    }
+
+    private void setupToolbar() {
+        LinearLayout root = (LinearLayout)findViewById(android.R.id.list).getParent().getParent().getParent();
```

Comment:
i don't have access to the layout in another way since this is a PreferencesActivity